modules/targetAcquisition/taxi/TaxiWorker.py

```diff
@@ -9,9 +30,12 @@ def taxi_worker(pause, exitRequest, pipelineIn, pipelineOut):
         pause.release()
         
         frame = pipelineIn.get()
-        taxi.main(frame)
-
```

Comment:
Pls remove all your changes from TaxiWorker, I'd recommend just unstaging it. Reason is that there's another PR I just merged that implements this with validation which is preferable, my apologies for assigning overlapping tasks.